lib/utils.js

```diff
@@ -57,25 +58,40 @@ async function getAndCheckIosSdkVersion () {
   }
 }
 
-function translateDeviceName (platformVersion, devName = '') {
-  let deviceName = devName;
-  switch (devName.toLowerCase().trim()) {
-    case 'iphone simulator':
-      deviceName = 'iPhone 6';
-      break;
-    case 'ipad simulator':
-      // iPad Retina is no longer available for ios 10.3
-      //   so we pick another iPad to use as default
-      deviceName = platformVersion && util.compareVersions(platformVersion, '<', '10.3') ? 'iPad Retina' : 'iPad Air';
-      break;
+/**
+ * Get the generic simulator for a given IOS version and device type (iPhone, iPad)
+ *
+ * @param {string|number} platformVersion IOS version. e.g.) 13.0
+ * @param {string} deviceName Type of IOS device. Can be iPhone, iPad (possibly more in the future)
+ */
+function getGenericSimulatorForIosVersion (platformVersion, deviceName) {
+  let genericSimulators = iosGenericSimulators[deviceName];
+
+  if (genericSimulators) {
+    genericSimulators = genericSimulators.sort(([simOne], [simTwo]) => util.compareVersions(simOne, '<', simTwo) ? -1 : 1);
+
+    // Find the highest iOS version in the list that is below the provided version
+    let genericIosSimulator;
+    for (const [platformVersionFromList, iosSimulator] of genericSimulators) {
+      if (util.compareVersions(platformVersionFromList, '>', platformVersion)) {
+        break;
+      }
+      genericIosSimulator = iosSimulator;
+    }
+    return genericIosSimulator;
   }
 
-  if (deviceName !== devName) {
-    log.debug(`Changing deviceName from '${devName}' to '${deviceName}'`);
-  }
   return deviceName;
 }
 
+function translateDeviceName (platformVersion, deviceName = '') {
+  const deviceNameTranslated = getGenericSimulatorForIosVersion(platformVersion, deviceName.toLowerCase().trim());
```

Comment:
is it necessary to change the device name to lower case?

Reply:
@mykola-mokhnach yeah if somebody provides `iPhone Simulator` we want it to be case-insensitive.